ibft/instance.go

```diff
@@ -134,9 +146,23 @@ func (i *Instance) Start(inputValue []byte) {
 	i.triggerRoundChangeOnTimer()
 }
 
-// Stage returns the instance message state
-func (i *Instance) Stage() proto.RoundState {
-	return i.State.Stage
+// Stop will trigger a stop for the entire instance
+func (i *Instance) Stop() {
+	i.stopLock.Lock()
+	defer i.stopLock.Unlock()
+
+	i.stop = true
+	i.stopRoundChangeTimer()
+	i.SetStage(proto.RoundState_Stopped)
+	i.Logger.Info("stopping iBFT instance...")
```

Comment:
Do we need to remove instance in ibft wrapper too?

Reply:
no, a stopped instance is also the current instance. 
I did find out I didn't set current instance so I've fixed it.